src/components/reactTasks/dislike.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+
+export default class Disike extends React.Component {
+  constructor() {
+    super()
+    this.state = {disliked: true}
+    this.handleClick = this.handleClick.bind(this)
+  }
+
+handleClick(event) {
```

Comment:
This is more what I was thinking! There is a much easier way to write what you have one below, using `!` as you did in the previous example. Take a look at that, and this ink to logical operators (you only need to look at the definition of `!`) https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_Operators

And you should see a much shorter way of writing this.